server/api_server/api_server_start.orig.py

```
@@ -0,0 +1,1135 @@
+import threading
+import sys
+import os
+
+from flask import Flask, request, jsonify, Response
+from models.device_instance import DeviceInstance  # noqa: E402
```

Ruff (0.14.11)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)